.github/workflows/integration-tests-smoke.yml

```diff
@@ -182,7 +163,7 @@ jobs:
         uses: ./.github/actions/install-cairo
       - name: Setup GitHub Token
         id: setup-github-token
-        uses: smartcontractkit/.github/actions/setup-github-token@9e7cc0779934cae4a9028b8588c9adb64d8ce68c # setup-github-token@0.1.2
+        uses: smartcontractkit/.github/actions/setup-github-token@ac467a8b9e64236300abf68f092dab11f67f499b # TODO
```

Comment:
I guess these two should be set to the most recent versions rather than latest head hash.